sphinx/texinputs/sphinx.sty

```diff
@@ -209,6 +209,61 @@
   \global\Sphinx@myfirstframedpassfalse
 }
 
+% For linebreaks inside Verbatim environment from package fancyvrb.
+\newbox\Sphinxcontinuationbox
+\newbox\Sphinxvisiblespacebox
+% These are user customizable e.g. from latex_elements's preamble key.
+% Use of \textvisiblespace for compatibility with XeTeX/LuaTeX/fontspec.
+\newcommand*\Sphinxvisiblespace
+                {\textcolor{red}{\textvisiblespace}}
```

Comment:
should we make this color customizable ? it is possible already for the user to use `'preamble'` key to do own definition to overwrite the one from `sphinx.sty`.